railties/lib/rails/test_unit/test_requirer.rb

```diff
@@ -15,7 +15,7 @@ def require_files(patterns)
       private
         def expand_patterns(patterns)
           patterns.map do |arg|
-            arg = arg.gsub(/:(\d+)?$/, '')
+            arg = arg.gsub(/(:\d*)+?$/, '')
```

Comment:
@kaspth Would there be an issue if we didn't change? Because with current regex, if user types by mistake  `foo.rb::67` then it runs test case defined at line number 67. I think this would be more useful. Thoughts please?